src/scripts/components/Resource.jsx

```diff
@@ -2,9 +2,43 @@ import cx from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import { ResourceRight, UtilitySystem } from '../components';
+import { ResourceBottom, ResourceRight, UtilitySystem } from '../components';
 
 class Resource extends React.Component {
+  getChildren = () => {
+    let returnChild = null;
+    const { children } = this.props;
+
+    return React.Children.map(children, (child) => {
+      if (!child) return false;
```

Comment:
I'm not sure if this check is needed (also below). The [documentation](https://reactjs.org/docs/react-api.html#reactchildren) says it will return `null` or `undefined` if children is `null` or `undefined`. If you do need this, maybe consider returning `null` like below rather than false.

Reply:
I think you can just remove that line unless when you call that function you are specifically looking for boolean `false` or `null`. It seems like `React.Children.map` will return `undefined` / `null` for you and if all you care about is that it's "falsey"...

Reply:
I made the `null` change, but it looks like taking that check out completely busts the app. It could possibly be when we're looking for properties on children that may not be child "components" and not have sub-properties we're checking against, but looks like it may be a good idea to keep-in.